### DISCUSSION TITLE
any support for completions related to orgmode

### DISCUSSION THREAD

**Discussion by Aneeqasif** [minad/marginalia#172] [Ideas]
like if there were any hints to what these flags are it would be very helpful , just as in other situations where marginalia provides hints 

![image](https://github.com/minad/marginalia/assets/78728701/e112a83a-ae04-4a36-8c2d-22b8eb8f13e2)

**Comment by minad** [comment]
Support for in-buffer completions is out of scope of Marginalia. Also adding such Org-specific functionality would not be a good fit here, it should better be provided by Org itself. As far as I know @karthink is working on improving annotations and generally completion in Org buffers. See https://github.com/minad/corfu/discussions/419 for a related discussion.

**Reply by Aneeqasif** [reply]
right
